R/zarr-array.R

```diff
@@ -1317,3 +1262,82 @@ ZarrArray <- R6::R6Class("ZarrArray",
 as.array.ZarrArray = function(x, ...) {
   x$as.array()
 }
+
+get_parallel_settings <- function(on_windows = (.Platform$OS.type == "windows"),
```

Comment:
```suggestion
get_parallel_settings <- function(on_windows = (.Platform$OS.type == "windows"),
```
Would you be able to add documentation comments to document the parameters and return value?

Reply:
For sure.